### PR TITLE
Initial yami-ichi update

### PR DESCRIPTION
**Pull request by ana0**
No bling or vendors yet, but in the interests of getting the vendor call off there as quickly as possible

### CODE REVIEW

**Comment by garrying** [comment]
@ana0 There's a line on the homepage, below the `Registration Details` button, that mentions applying to vend. Would it make sense to rephrase it to attending the Yami-Ichi?

**Comment by dcwalk** [comment]
Hey @garrying  I tried to update index.
Can you give this a quick look? I'll aim to merge around lunch and send the emails out. Thanks @ana0 !

**Comment by ana0** [comment]
Moved the link in index, and removed another mention of vending in about

**Review by garrying — APPROVED** [review]
👍
